shared/ff_util/include/ff_util/ff_names.h

```diff
@@ -95,7 +95,9 @@
 #define SUBSYSTEM_COMMUNICATIONS                    "comm"
 
 #define NODE_DDS_ROS_BRIDGE                         "dds_ros_bridge"
+#define NODE_ASTROBEE_ASTROBEE_BRIDGE               "astrobee_astrobee_bridge"
 
+#define SERVICE_COMMUNICATIONS_ASTROBEE_ASTROBEE_BRIDGE_TRIGGER "comm/astrobee_astrobee_bridge/trigger"
```

Comment:
I'm not sure that I like the name trigger since it isn't clear what is being triggered unless you look at the code. I would 
name it something like start_comms.